packages/build/src/plugins/child/main.js

```diff
@@ -1,7 +1,10 @@
+import '@netlify/opentelemetry-sdk-setup/bin.js'
+import { stopTracing } from '@netlify/opentelemetry-sdk-setup'
```

Comment:
I just remembered why this isn't feasible @lukasholzer @pieh. This implies `@netlify/build` needs to depend on `@netlify/opentelemetry-sdk-setup` which isn't the case (deliberatly - https://linear.app/netlify/issue/COM-110/make-opentelemetry-an-optional-dependency-in-build), it's currently setup as an optional dep which will exist in production but not through other `@netlify/build` consumption methods (e.g. CLI) meaning plugin executions would break:
- https://github.com/netlify/build/blob/c5ab0db977e90b92e29ab0a942bc8fc25d091f91/packages/build/package.json#L159-L163

Can we investigate what's holding us back from using the preload mechanism? I was also thinking that that would allow us to ensure proper `nodejs` compatibility for the SDK setup, making sure we only run the setup if the plugins are being executed with a compatible node version (thereby not adding more stuff to the pile of compatibility chaos we already need to maintain)